runtime/kusama/src/governance/tracks.rs

```diff
@@ -85,8 +85,8 @@ const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 15
 		pallet_referenda::TrackInfo {
 			name: "whitelisted_caller",
 			max_deciding: 10,
-			decision_deposit: 10_000 * GRAND,
-			prepare_period: 3 * HOURS,
+			decision_deposit: 100 * GRAND,
```

Comment:
thank you for this. But doesn't this setup now mean that one can maliciously jam this track with just 10*100 GRAND, which is only 3.33k KSM. Or is there a "decision gate" requiring the whitlelisting to be done to enter deciding?

Reply:
The `root` track would have the same issue then. I wonder if the better option is to set the Root track to a high decision deposit and also allow the `WhitelistOrigin` to remove proposals from the `WhitelistedCaller` track, essentially saying "we're not going to Whitelist this call so there's no need for a Whitelist track referendum".

Reply:
Updated `QUID` and increased the `max_deciding` so this is now 100 kKSM to jam it. With `ReferendumKiller` origin, we can remove spam and slash the spammer's deposits.